inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp

```diff
@@ -20,6 +20,9 @@ std::vector<std::string> disabledTestPatterns() {
         // TODO: Issue 32023
         R"(.*(QuantGroupConvBackpropData2D)*QG=Pertensor.*)",
         // TODO: Issue 31845
-        R"(.*(FakeQuantize).*)"
+        R"(.*(FakeQuantize).*)",
+        // TODO: Issue: 32521
+        R"(.*(EltwiseLayerTest).*eltwiseOpType=Sub.*secondaryInputType=CONSTANT.*netPRC=FP16.*)",
+        R"(.*(EltwiseLayerTest).*IS=.*1.1.1.1.*secondaryInputType=PARAMETER.*opType=SCALAR.*)"
```

Comment:
change to:
`R"(.*(EltwiseLayerTest).*IS=(.*\..*\..*\..*\..*\..*)_eltwise.*secondaryInputType=PARAMETER.*opType=SCALAR.*)"`
this will cover all broadcast cases where blob dims > 5 which is exactly what is causing failures